### PR TITLE
chore: Update to Node 16 and sss-wasm

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: ceremony.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-const { CeramicClient } = require('@ceramicnetwork/http-client')
-const { TileDocument } = require('@ceramicnetwork/stream-tile')
-const { Ed25519Provider } = require('key-did-provider-ed25519')
-const KeyResolver = require('key-did-resolver')
-const seedsplit = require('seedsplit')
-const bip39 = require('bip39')
+import { DID } from 'dids'
+import { CeramicClient} from '@ceramicnetwork/http-client'
+import { TileDocument} from "@ceramicnetwork/stream-tile";
+import { Ed25519Provider } from "key-did-provider-ed25519";
```
Comment (oed) [inline ceremony.js]:
```suggestion
import { TileDocument} from "@ceramicnetwork/stream-tile"
import { Ed25519Provider } from "key-did-provider-ed25519"
```
plz

File: ceremony.js [hunk 2]
```diff
@@ -121,8 +121,19 @@ function prompt(question) {
     stdin.resume()
     stdout.write('\n' + question)
 
-    stdin.on('data', data => resolve(data.toString().trim()))
-    stdin.on('error', err => reject(err))
+    const stdinHandler = (data) => {
+      stdin.off('data', stdinHandler)
+      stdin.off('error', errorHandler)
```
Comment (oed) [inline ceremony.js]: Why turn these of?

**Review by oed — COMMENTED** [review]
One question

File: ceremony.js (hunk 1 above)
Comment (ukstv) [reply ceremony.js]: Prettified.

**Review by oed — APPROVED** [review]

**Pull request merged**